_posts/2017-10-05-Kickstarter-Trajectory-2017.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+authors: [nick_geudens,ken_de_becker,michiel_willems,jef_willems,simon_buysse]
+title: "Kickstarter Trajectory 2017"
+image: /img/kicks.png
+tags: [Spring, Angular2, Unit Test, Microservices, Spring Boot, Ionic, MongoDB]
+category: Kickstarters
+comments: true
+---
+
+>This year, five talented people started the Ordina Kickstarter trajectory.
+Each of them looking for a challenge and a fresh start!
+For some, it was a transition between school and work, and Ordina handled this very well.
+The main goal of this trajectory was to widen every student’s knowledge of the IT world.
+They taught us the basics of every topic that’s hot at the moment.
+This will definitely come in handy during our first project.
+
+
+# First impressions
+
+When we arrived the first day, we were welcomed with breakfast and afterwards, we got a tour around the building.
+We immediately felt at home as everybody is really friendly and helpful.
+The corporate culture here is truly a plus for Ordina.
+We also got our equipment which consisted of a car and a laptop.
+Some people had a preference for Mac, which was no problem at all.
+From day one Ordina makes sure that their consultants are well equipped to work successfully.
+
+# What is the Kickstarter trajectory?
+
+The Kickstarter trajectory consisted of intensive training spanning two months.
+During this time, courses on different frontend and backend technologies, methods, tools and soft skills were given by senior consultants and external lecturers. 
+This trajectory is ideal for graduating students and people who want to make a switch to IT.
+The constant guidance and support made this a good preparation for our first project.
+
+During the fourth week we had a team building together with all the kickstarters from the different units. 
```

Comment:
Call it a "team building event" as I'm sure you didn't have a building for your team. You can also drop the "together"